packages/account/src/configs.ts

```diff
@@ -1,4 +1,6 @@
-export const FUEL_NETWORK_URL =
+// #region FUEL_NETWORK_URL
```

Comment:
Duplication from #1949, is one based off the other?

Reply:
https://github.com/FuelLabs/fuels-ts/commit/d47ec8d527c29efc017ffa24a30d2ceeac896332

Removed these changes.